life/updated/03-testimonies.adoc

```diff
@@ -76,61 +76,61 @@ John Thornton
 She received the truth in the love of it in her young years,
-as I have heard from her own mouth, during the time of my acquaintance with her,
+as I have heard from her own mouth during the time of my acquaintance with her,
 which was about fifteen years.
 She lived about seven miles from us, and sometimes came to visit our meeting at Watford,
 and I with many more were glad to see her,
 she being one on whom the Lord was pleased to bestow a gift of the ministry,
 that she might tell unto others what God had done for her soul.
 My heart has been well affected many times with her testimony,
 for she spoke as one having authority, almost to admiration,
-considering the weakness of body with which she was afflicted; but the Lord was with her,
-who gives strength to the weak, and his power was her support.
-It was her lot many times when she came to our meeting, to lodge at my house,
-my dear mother, Hannah Neale, and she being very intimate friends,
-and heartily loving each others company, which I also loved, for it was pleasant,
+considering the weakness of body with which she was afflicted. But the Lord was with her,
+who gives strength to the weak, and His power was her support.
+It was her lot many times when she came to our meeting to lodge at my house,
+my dear mother, Hannah Neale, and she being very intimate friends
+and heartily loving each others`' company, which I also loved, for it was pleasant,
 and her discourse was edifying to me.
-I have been affected to hear her give an account of the sufferings,
+I have been affected to hear her give an account of the sufferings
 which she and many others went through in early days.
 When they were going to meeting, they went in peril of their lives,
 the wicked and ungodly people throwing stones, clods, and other things at them.
 And although many rose up against them,
-yet how wonderfully did the Lord preserve his people through many tribulations;
+yet how wonderfully did the Lord preserve His people through many tribulations,
 most of which, blessed be the Lord, we in this day are free from.
 
-Much might be said concerning our friend: she was a valiant woman for truth on earth,
-a mother in Israel, and a worthy faithful elder in the church of Christ in her time.
-And my desire is, that we who are of a younger generation, and are yet left behind,
+Much might be said concerning our friend. She was a valiant woman for truth on earth,
+a mother in Israel, and a worthy, faithful elder in the church of Christ in her time.
+And my desire is that we who are of a younger generation, and are yet left behind,
 may in our measure be found treading in the steps of our ancients,
 who have served the Lord faithfully in their generation, and are removed from us.
-Though our loss of such worthies be great;
-yet doubtless they are entered into those mansions of eternal bliss,
-where joy unspeakable and full of glory is their portion, forevermore.
+Though our loss of such worthies be great,
```

Comment:
maybe "is" great. (these are examples of the SUBJUNCTIVE in older English. It is now more common to leave out the subjunctive "be" and just say "is")